.github/workflows/ci.yaml

```diff
@@ -33,10 +33,11 @@ jobs:
           - "3.10"
           - "3.11"
           - "3.12"
+          - "3.13"
```

Comment:
Looks like we're not 3.13 compatible yet (because of OpenMM 8.2)

Reply:
Yep! Just wanted to check. Removing now.